Userland/Utilities/sizefmt.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2024, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/CharacterTypes.h>
+#include <AK/Checked.h>
+#include <AK/NumberFormat.h>
+#include <AK/Optional.h>
+#include <AK/StringView.h>
+#include <AK/Types.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+
+static ErrorOr<u64> use_integer_with_char_suffix(StringView argument, size_t suffix_length)
+{
+    if (suffix_length >= argument.length())
+        return Error::from_string_literal("Invalid value was specified");
+    argument = argument.substring_view(0, argument.length() - suffix_length);
+
+    Optional<u64> numeric_optional = argument.to_number<u64>();
+    if (numeric_optional.has_value())
+        return numeric_optional.release_value();
+    return Error::from_string_literal("Invalid value was specified");
+}
+
+static ErrorOr<u64> handle_char_suffix(char suffix, AK::HumanReadableBasedOn human_readable_based_on)
+{
+    u64 suffix_multiplier = 1;
+    switch (suffix) {
+    case 'k':
+    case 'K':
+        suffix_multiplier = human_readable_based_on == AK::HumanReadableBasedOn::Base2 ? KiB : KB;
```

Comment:
```suggestion
        suffix_multiplier = (human_readable_based_on == AK::HumanReadableBasedOn::Base2 ? KiB : KB);
```

Might be personal preference, and the compiler might be yelling, but it does provide additional guidance for differentiating between the assignment operator and the rest.

Same applies for the other cases.